compiler/circle-tensordump/driver/Driver.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Dump.h"
+
+#include <foder/FileLoader.h>
+
+#include <functional>
+#include <iostream>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+using OptionHook = std::function<std::unique_ptr<circletensordump::DumpInterface>(void)>;
+
+int entry(int argc, char **argv)
+{
+  if (argc < 3)
+  {
+    std::cerr << "ERROR: Failed to parse arguments" << std::endl;
+    std::cerr << std::endl;
+    std::cerr << "USAGE: " << argv[0] << " [options] [circle]" << std::endl;
+    std::cerr << "   --tensors : dump tensors in circle file" << std::endl;
+    std::cerr << "   --tensors_to_hdf5 : dump tensors in circle file to hdf5 file" << std::endl;
```

Comment:
Instruction message should be revised (--tensors_to_hdf5 option requires two arguements). 

./circle-tensordump --tensors_to_hdf5 </path/to/input/circle> </path/to/output/h5>